.travis.yml

```diff
@@ -2,31 +2,85 @@ language: python
 sudo: false
 
 python:
-  - "2.7"
+- 2.7
 
 env:
-  - TOXENV=flake8
-  - TOXENV=docs
-  - TOXENV=assets
-  - TOXENV=es
-  - TOXENV=addons
-  - TOXENV=devhub
-  - TOXENV=editors
-  - TOXENV=amo
-  - TOXENV=users
-  - TOXENV=main
```

Comment:
You said to make the DISPLAY env variable global. I don't know of a way to make one env global, as travis would give me an error if I just added ```global``` to the env list.